.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 # Please follow the general troubleshooting steps first:
 
-- [ ] Has there been no other issue like this before?
+- [ ] Has there been no other similar issue like this one already reported?
```

Comment:
```suggestion
- [ ] Was there no similar issue like this one reported?
```
technically the like this is redundant too, as the reader already sees which issue this is referring to.